projects/cm_mcu/AlarmUtilities.c

```diff
@@ -138,4 +138,163 @@ struct GenericAlarmParams_t tempAlarmTask = {
     .checkStatus = &TempStatus,
     .errorlog_registererror = &TempErrorLog,
     .errorlog_clearerror = &TempClearErrorLog,
+    .stack_size = 176,
+};
+
+///////////////////////////////////////////////////////////
+//
+// Voltage Alarms
+//
+///////////////////////////////////////////////////////////
+
+// current value of the thresholds
+#define INITIAL_ALARM_VOLT_PERCENT 0.05f // +/-5% from the ADC thresholds
+static float alarmVolt = INITIAL_ALARM_VOLT_PERCENT;
+
+float getAlarmVoltage(void)
+{
+  return alarmVolt;
+}
+void setAlarmVoltage(float voltthres)
+{
+  alarmVolt = voltthres;
+}
+
+// current status of voltages
+static uint8_t currentVoltStatus[3] = {0.0, 0.0, 0.0}; // FIXME : bitmasks for voltage statuses
+
+// Status flags of the voltage alarm task
+static uint32_t status_V = 0x0;
+// read-only, so no need to use queue
+uint32_t getVoltAlarmStatus()
+{
+  return status_V;
+}
+// check the current voltage status.
+// returns +1 for warning, +2 or higher for error
+int VoltStatus()
+{
+  bool f1_enable = isFPGAF1_PRESENT();
+  bool f2_enable = isFPGAF2_PRESENT();
+#ifndef REV2                         // REV1
+  uint8_t GENFPGA_VOLTAGE_MASK = 63; // 0b111111 by default
```

Comment:
When doing bitmasks it is convention to use hex not decimal

Reply:
Done.